src/Makefile

```diff
@@ -3,6 +3,8 @@
 obj-m := elastio-snap.o
 elastio-snap-objs:= main.o
 
+NETLINK_DEBUG=y
```

Comment:
Is it true by default?
May be 
`NETLINK_DEBUG ?= y`
For override by env variable on build?